pyqir/src/builder.rs

```diff
@@ -359,6 +359,36 @@ impl Builder {
         };
         unsafe { Value::from_raw(py, owner, value) }
     }
+
+    /// The ‘zext’ instruction zero extends its operand to the given type.
+    ///
+    /// :param Value val: Value to be converted.
+    /// :param Type ty: Target type.
+    /// :returns: The result.
+    /// :rtype: Value
+    #[pyo3(text_signature = "(self, val, ty)")]
+    fn zext(&self, py: Python, val: &Value, ty: &Type) -> PyResult<PyObject> {
+        let owner = Owner::merge(py, [&self.owner, val.owner(), ty.owner()])?;
```

Comment:
Type doesn't have an owner.

Reply:
Done in [7e221c8](https://github.com/qir-alliance/pyqir/pull/291/commits/7e221c8bfc52745e18ff5d16b725e2c9ca23aaa9)